src/main/java/com/ezasm/simulation/Simulator.java

```diff
@@ -69,6 +71,7 @@ public void resetAll() {
         labels.clear();
         transforms.clear();
         initialize();
+        Window.getInstance().getRegisterTable().removeHighlightValue();
```

Comment:
This code should not be in the Simulator class, it should only be in GUI elements that cause a `reset` on the simulator (e.g., pressing the `reset` button).